dbms/src/Functions/FunctionsComparison.h

```diff
@@ -342,8 +356,8 @@ struct StringComparisonWithCollatorImpl
     }
 
     static void constantConstant(
-        const std::string & a,
-        const std::string & b,
+        std::string_view a,
```

Comment:
Why not const ref?

Reply:
`std::string_view` only contain a pointer and a size_t (16 bytes totally). It's fine to use const ref or object itself.

Reply:
Fixed